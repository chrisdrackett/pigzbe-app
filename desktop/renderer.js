import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
