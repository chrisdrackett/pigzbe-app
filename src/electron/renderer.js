import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
