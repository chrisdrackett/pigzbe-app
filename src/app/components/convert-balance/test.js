import React from 'react';
import renderer from 'react-test-renderer';
import ConvertBalance from './';

const coins = ['XLM', 'BTC', 'ETH', 'EUR', 'USD', 'JPY', 'GBP'];
const dps = {XLM: 7, BTC: 8, ETH: 8, EUR: 2, USD: 2, JPY: 0, GBP: 2};

const props = {
    exchange: {XLM: 1, BTC: 0.00004463, ETH: 0.0006136, EUR: 0.3459, USD: 0.4183, JPY: 47.3, GBP: 0.3023},
    balance: '1223.00',
    coins,
    dps,
};

describe('ConvertBalance', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<ConvertBalance {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
