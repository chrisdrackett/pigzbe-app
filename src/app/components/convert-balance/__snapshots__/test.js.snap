// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConvertBalance renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "rgb(247, 247, 250)",
      "flex": 1,
      "justifyContent": "center",
      "paddingBottom": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "rgb(74, 74, 74)",
        "fontFamily": "Poppins",
        "fontSize": 16,
        "marginTop": 25,
      }
    }
  >
    Your balance converted
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}
    style={
      Object {
        "color": "rgb(74, 74, 74)",
        "fontFamily": "Poppins",
        "fontSize": 12,
      }
    }
  >
    Prices by CryptoCompare
  </Text>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "alignSelf": "stretch",
        "flex": 1,
        "flexDirection": "row",
        "flexWrap": "wrap",
        "justifyContent": "space-between",
        "marginTop": 20,
        "paddingLeft": 20,
        "paddingRight": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-xlm.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        XLM
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        1,223
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-btc.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        BTC
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        0.05458249
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-eth.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        ETH
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        0.7504328
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-eur.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        EUR
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        423.04
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-usd.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        USD
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        511.58
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-jpy.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        JPY
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        57,848
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexShrink": 0,
          "justifyContent": "center",
          "marginBottom": 20,
          "minWidth": "26%",
          "width": "26%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/coin-icon/images/coin-gbp.png",
          }
        }
        style={
          Array [
            Object {
              "height": 35,
              "width": 35,
            },
            Object {
              "marginBottom": 15,
            },
          ]
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgb(74, 74, 74)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        GBP
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(74, 74, 74, 0.5)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
          }
        }
      >
        369.71
      </Text>
    </View>
  </View>
</View>
`;
